contracts/TitleEscrow.sol

```diff
@@ -110,4 +114,30 @@ contract TitleEscrow is Context, ITitleEscrow, HasNamedBeneficiary, HasHolder, E
     emit TitleCeded(address(tokenRegistry), newBeneficiary, _tokenId);
     tokenRegistry.safeTransferFrom(address(this), address(newBeneficiary), _tokenId);
   }
+
+  function deployNextEscrow(address beneficiary, address holder) public returns (address) {
+    return tokenRegistry.deployTitleEscrow(beneficiary, holder);
```

Comment:
Hmmm thinking if we should have a global factory on each network instead of relying on the registry contract. When this contract is instantialise/constructed it can be passed in the factory,

Moving forward, I'm thinking if we have a global ProxyFactory, we can then "copy" itself using the ProxyFactory .

Reply:
maybe skip the global factory and go straight to proxy because i don't want to have to keep updating the global factory while we're still in development with lots of updates, wdyt?

Reply:
Nah i think we can leave this, just food for thoughts.